fetch.bs

```diff
@@ -3898,20 +4098,35 @@ these steps:
 
  <li><p>Let <var>actualResponse</var> be null.
 
+ <li><p>Let <var>timingInfo</var> be <var>fetchParams</var>'s<a for="fetch params">timing
+ info</a>.
+
  <li>
   <p>If <var>request</var>'s <a>service-workers mode</a> is "<code>all</code>", then:
 
   <ol>
    <li><p>Let <var>requestForServiceWorker</var> be a <a for=request>clone</a> of
    <var>request</var>.
 
+   <li><p>Set <var>fetchParams</var>'s <a for="fetch params">timing info</a>'s
+   <a for="fetch timing info">worker start time</a> to the <a for=/>unsafe shared current time</a>.
+
    <li><p>Set <var>response</var> to the result of invoking <a for=/>handle fetch</a> for
    <var>requestForServiceWorker</var>. [[!HTML]] [[!SW]]
 
    <li>
     <p>If <var>response</var> is not null, then:
 
     <ol>
+     <li><p>Set <var>fetchParams</var>'s <a for="fetch params">timing info</a>'s
+     <a for="fetch timing info">response start time</a> to the
+     <a for=/>unsafe shared current time</a>.
```

Comment:
@wanderview - Can you verify that this matches what's currently implemented?

Reply:
I updated this. Response start time is only available when dealing with actual HTTP responses. A response from a SW can be an actual HTTP response, in which case it will mark a `response start` at the appropriate time.